build_system/tdl-config.cmake

```diff
@@ -166,8 +122,9 @@ file (WRITE "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/src.cxx" "${CX
 try_compile (TDL_PLATFORM_TEST
              ${CMAKE_BINARY_DIR}
              ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/src.cxx
-             CMAKE_FLAGS         "-DCOMPILE_DEFINITIONS:STRING=${CMAKE_CXX_FLAGS} ${TDL_CXX_FLAGS}"
-                                 "-DINCLUDE_DIRECTORIES:STRING=${CMAKE_INCLUDE_PATH};${TDL_INCLUDE_DIR};${TDL_DEPENDENCY_INCLUDE_DIRS}"
+             CXX_STANDARD        17
+             CMAKE_FLAGS         "-DCOMPILE_DEFINITIONS:STRING=${CMAKE_CXX_FLAGS}"
+                                 "-DINCLUDE_DIRECTORIES:STRING=${CMAKE_INCLUDE_PATH};${TDL_INCLUDE_DIR}"
```

Comment:
why did you remove the `TDL_CXX_FLAGS` ?

Reply:
I don't see where it is required/needed. (Might have some deeper meaning I don't know of, but it didn't appear when integrating in OpenMs nor sharg-parser